language/tools/test_generation/tests/struct_instructions.rs

```diff
@@ -124,44 +141,141 @@ fn bytecode_unpack_signature_not_satisfied() {
 
 #[test]
 fn bytecode_unpack() {
-    let module: CompiledModuleMut = generate_module_with_struct();
+    let module: CompiledModuleMut = generate_module_with_struct(false);
+    let mut state1 = AbstractState::from_locals(module, HashMap::new());
+    let struct_value = create_struct_value(&state1.module);
+    state1.stack_push(struct_value.clone());
+    let state2 = common::run_instruction(
+        Bytecode::Unpack(StructDefinitionIndex::new(0), LocalsSignatureIndex::new(0)),
+        state1,
+    );
+    if let SignatureToken::Struct(_, tokens) = struct_value.token {
+        assert_eq!(
+            state2.stack_len(),
+            tokens.len(),
+            "stack type postcondition not met"
+        );
+    } else {
+        panic!("Created struct is malformed");
```

Comment:
It seems like bad style to panic inside a test. Instead just use assert!(false, ...)

Reply:
I will update this in the next PR.